packages/message-manager/src/AbstractMessageManager.test.ts

```diff
@@ -120,6 +120,28 @@ describe('AbstractTestManager', () => {
     expect(message.rawSig).toBe('rawSig');
   });
 
+  it('should set message to one of the allowed statuses', () => {
```

Comment:
Just for the heads up
https://github.com/MetaMask/core/pull/1139#discussion_r1145123454